### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cfauci**
This is on a branch called README and a subbranch with my name. Just some minor updates this round for the first reviewers first comment

# Description

[Description]

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +31,124 @@ go install  ./...
...
+)
+
+functionForTheTool(arguments){
+ bedRecords := bed.Read(bedFileFromArguments)
+}
+```
+
+Let's say you wanted to use this command to read in a bed file and create an amino acide code from each of its regions. That function may look something like this:
+
+```
```
Comment (edotau) [inline README.md:93]: when opening a block of code/text in this way it's good practice to specify what language or type of text the block the upcoming lines are. the markdown format will be able to detect coloring themes as a result. some other key words to add here are: `text, csv, python, golang, etc., ...

```suggestion
```golang
```

File: README.md [hunk 2]
```diff
@@ -30,6 +31,124 @@ go install  ./...
...
+
+- `FuncToDoThings` not `funcToDoThings` in its package
+
+Generally, in gonomics we have a package for a file type where we have an implementation as a struct in our code base. So if you plan to read in a file, look for a package in gonomics with that file type as its name. 
+
+- `gonomics/bed` if you wanted to use in a bed file.
+
+These packages will have read and write functions as well as some other useful tools within them. All you would have to do from there to read in a bed file is import the package into your newTool.go file in the command directory.
+
+```
```
Comment (edotau) [inline README.md:77]:
```suggestion
```golang
```

File: README.md [hunk 3]
```diff
@@ -30,6 +31,124 @@ go install  ./...
...
+
+functionForTheTool(arguments){
+ bedRecords := bed.Read(bedFileFromArguments)
+}
+```
+
+Let's say you wanted to use this command to read in a bed file and create an amino acide code from each of its regions. That function may look something like this:
+
+```
+##gonomics/cmd/newTool.go
```
[the remark this answers is not in the document]
Comment (cfauci) [reply README.md:94]: Thank you for the improvements, Eric!

File: README.md [hunk 4]
```diff
@@ -30,6 +31,124 @@ go install  ./...
...
+- `FuncToDoThings` not `funcToDoThings` in its package
+
+Generally, in gonomics we have a package for a file type where we have an implementation as a struct in our code base. So if you plan to read in a file, look for a package in gonomics with that file type as its name. 
+
+- `gonomics/bed` if you wanted to use in a bed file.
+
+These packages will have read and write functions as well as some other useful tools within them. All you would have to do from there to read in a bed file is import the package into your newTool.go file in the command directory.
+
+```
+##gonomics/cmd/newTool.go
```
Comment (edotau) [inline README.md:78]:
```suggestion
// gonomics/cmd/newTool.go
```

File: README.md [hunk 5]
```diff
@@ -30,6 +31,124 @@ go install  ./...
...
+import (
+"github.com/vertgenlab/gonomics/bed"
+)
+
+functionForTheTool(arguments){
+ bedRecords := bed.Read(bedFileFromArguments)
+}
+```
+
+Let's say you wanted to use this command to read in a bed file and create an amino acide code from each of its regions. That function may look something like this:
```
Comment (hcnapier) [inline README.md:91]: tiny typo: "acid" instead of "acide"

**Review by RavenYL — DISMISSED** [review]

**Review by edotau — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -30,6 +30,10 @@ go install  ./...
 
 Executables will be present in Go binary folder (`~/go/bin` by default)
```
[the remark this answers is not in the document]
Comment (craiglowe) [reply README.md:31]: Maybe we can also add a sentence that running the command with no arguments prints the usage.  I realize that is also in the wiki, but maybe it is a short thing to include here as well.

**Review by edotau — APPROVED** [review]

**Review by craiglowe — APPROVED** [review]
Great!

**Pull request merged**